apps/frontend/src/pages/route-description/styles.module.css

```
@@ -32,3 +33,22 @@
 	margin: 0;
 	overflow-wrap: break-word;
 }
+
+.header-container {
+	display: flex;
+	column-gap: 20px;
```

let's use 16 for small gaps
```suggestion
	column-gap: 16px;
```